hw/dv/dpi/spidpi/spidpi.c

```diff
@@ -132,7 +132,7 @@ char spidpi_tick(void *ctx_void, const svLogicVecVal *d2p_data) {
       }
     }
   }
-  // SPI clock toggles every 4th tick (i.e. freq=master/8)
+  // SPI clock toggles every 4th tick (i.e. freq=primary-frequency/8)
```

Comment:
Maybe this should be an underscore? Otherwise it looks like a subtraction if the reader is in "C mode".